src/commands/chains.ts

```diff
@@ -3,16 +3,38 @@ import { GlobalOptions } from "../cli.js";
 import { getChains } from "../utils.js";
 import type yargs from "yargs";
 
-export type Options = GlobalOptions;
+export interface Options extends GlobalOptions {
+  format: "pretty" | "json" | "jsonl";
+}
 
 export const command = "chains";
 export const desc = "List information about supported chains";
 
 export const builder: CommandBuilder<{}, Options> = (yargs) =>
-  yargs as yargs.Argv<Options>;
+  yargs.option("format", {
+    type: "string",
+    choices: ["pretty", "json", "jsonl"] as const,
+    description: "Output format. One of 'pretty', 'json', or 'jsonl'.",
+    default: "json",
+  }) as yargs.Argv<Options>;
 
 const chains = getChains();
 
 export const handler = async (_argv: Arguments<Options>): Promise<void> => {
+  const { format } = _argv;
+
+  if (format === "pretty") {
+    console.log(JSON.stringify(chains, null, 4));
+    return;
+  }
+  if (format === "jsonl") {
```

Comment:
There isn't a specific need for this, returning jsonl will enable users to pipe the output into commands that expect to be able to process data line by line.  more info: https://jsonlines.org/